src/actions/index.js

```diff
@@ -292,6 +294,15 @@ export const walletReloadData = () => ({
   type: types.WALLET_RELOAD_DATA,
 });
 
+export const storeRouterHistory = (routerHistory) => ({
```

Comment:
Missing parameter docstring

Reply:
Added in bb823f3176d1a311548edc28bf12922d331e65ce